runtime/doc/usr_05.txt

```diff
@@ -10,7 +10,7 @@ make Vim start with options set to different values.  Add plugins to extend
 Vim's capabilities.  Or define your own macros.
 
 |05.1|	The vimrc file
-|05.2|	The example vimrc file explained
+|05.2|	Example vimrc commands
```

Comment:
```suggestion
|05.2|	Example vimrc contents
```
as "vimrc commands" are not really a thing? (There's enough confusion of vimscript and ex commands; no reason to add another term to the mix...)

Reply:
(My concern was only with the specific term "vimrc commands", which could be misinterpreted as a specific kind of command.)

Reply:
> Example commands in a vimrc

?

Reply:
Also good!